screens/addTag/addTagComponents/LitterTagsCard.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions } from 'react-native';
+import { Colors, Body, Caption } from '../../components';
+
+const { width: SCREEN_WIDTH } = Dimensions.get('window');
+
+/**
+ * component to show added tags on image when AddTags screen is opened
+ * shown below Litter Status card
+ *
+ * Loop over Array of keys of tag Object to get Categories
+ * Then loop again Key of categories Object to get individual tag and its quantity
+ *
+ * tag : {
+ * "category" : {
+ *          "tag1" : quantity
+ *          }
+ * }
+ * @param {Object} tag -- > Object of tags
+ * @param {String} lang --> Selected Global Language
+ * @returns
+ */
+const LitterTagsCard = ({ tags, lang }) => {
+    return (
+        <>
+            {/* Only show if atleast one tag is present */}
+            {Object.keys(tags).length !== 0 && (
+                <View style={styles.card}>
+                    <Caption>Tags</Caption>
+
+                    <View style={{ marginTop: 8 }}>
+                        {Object.keys(tags).map(category => {
+                            return (
+                                <View>
+                                    <Body
+                                        dictionary={`${lang}.litter.categories.${category}`}
+                                    />
+                                    <View
+                                        style={{
+                                            flexDirection: 'row',
+                                            flexWrap: 'wrap'
+                                        }}>
+                                        {Object.keys(tags[category]).map(
```

Comment:
Do we need to check if tags[category] exists first?

Reply:
No need, This only executes if Tags are present... And category comes from the array of keys of tags object.